### ISSUE TITLE
automatic announcement of frame update

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#8345]
### Steps to reproduce:
Beforehand I apologize for the lack of technical elements to describe.
I would suggest for nvda to announce the update of frames, (at least some).
Example:
Open google and do a search in another language.
Example:
My google is in Portuguese, I did a search for:
screen reader
Google will show me a page with the results in English that is not my native language, because the term of my search was in English.
Now move to a result in another language, other than your native configured in google in my case I chose the page of the
Wikipedia in English.
below the button more details should be the link:
translate this page.
Open this link, it should open a page of google translator with the page translated.
Now on the google translator page there is a frame that contains the translated page.
When the page to be translated this frame takes to update with the translation of the page.

### Expected behavior:
Nvda should announce that the frame was updated automatically when the translated page is displayed.
I have seen other pages with the same behavior.
I saw another screen reader announcing this update.

### Actual behavior:
When the frame is updated with the translated page, nvda does not say anything, it is necessary to walk to the frame to see if the translation is ready.

### System configuration:
NVDA version:
next 15138

NVDA Installed or portable:
instaled
Other information:
> Example: Running in a VM

Windows version:
windows 7 e 10

Name and version of other software in use when reproducing the issue:
firefox 60
### Other questions:

Does the issue still occur after restarting your PC?
yes
Have you tried any other versions of NVDA?
> Please list them and the result
yes

**Comment by Adriani90** [comment]
In Firefox 126, NVDA reports "translation" and the title of the page in the translated language. So it works as expected with NVDA 2024.2 Beta 2.

In Chrome and MS Edge this doesn't work yet, I think someone needs to report this issue to Chromium as well.
